wrappers/nodejs/examples/glfw-window.js

```diff
@@ -6,6 +6,35 @@
 
 const glfw = require('node-glfw-3');
 const rs2 = require('../index.js');
+const now = require('performance-now');
```

Comment:
Is this dependency already in package.json?

Reply:
I missed it, will add it.